child_compassion/models/project_reservation.py

```diff
@@ -23,6 +23,7 @@ class ProjectReservation(models.Model):
     channel_name = fields.Char()
     icp_id = fields.Many2one(
         'compassion.project', 'Project', required=True)
+    global_id = fields.Char(default='')
```

Comment:
Instead of global_id, you should link to the child directly `child_id = fields.Many2one(...`